### PR TITLE
feat: stop idle playback #815

### PR DESCRIPTION
**Pull request by FrolYo**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Services/IdleAudioMonitor.cs [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    private async Task Monitor(
+        ChatId chatId,
+        Func<ChatId, State, Task> callback,
+        Options options,
+        CancellationToken cancellationToken)
+    {
+        await Task.Yield();
+        var clock = Clocks.SystemClock;
+        var recordingStartedAt = clock.Now;
+        await callback(chatId, new (false, null));
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Services/IdleAudioMonitor.cs:85]: Can be replaced with State.NotIdle?

File: src/dotnet/Audio/AudioSettings.cs [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+﻿namespace ActualChat.Audio;
 
 public sealed class AudioSettings
 {
     public string Redis { get; set; } = "";
     public string ServiceName { get; set; } = "actual-chat-app-service";
     public string Namespace { get; set; } = "default";
-    public TimeSpan IdleRecordingTimeout { get; set; } = TimeSpan.FromSeconds(3 * 60);
-    public TimeSpan IdleRecordingTimeoutBeforeCountdown { get; set; } = TimeSpan.FromSeconds(2.5 * 60);
+    public TimeSpan IdleRecordingTimeout { get; set; } = TimeSpan.FromSeconds(1.5 * 60);
```
[the remark this answers is not in the document]
Comment (FrolYo) [reply src/dotnet/Audio/AudioSettings.cs]: done

File: src/dotnet/Chat.UI.Blazor/Services/IdleAudioMonitor.cs (hunk 1 above)
Comment (FrolYo) [reply src/dotnet/Chat.UI.Blazor/Services/IdleAudioMonitor.cs]: fixed, thanks

File: src/dotnet/Chat.UI.Blazor/Services/IdleAudioMonitor.cs [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+        if (_isDisposed)
+            throw StandardError.Internal("Unable to start monitoring: IdleAudioMonitor is disposed");
+    }
+
+    public record Options(
+        TimeSpan IdleTimeout,
+        TimeSpan IdleTimeoutBeforeCountdown,
+        TimeSpan CheckInterval) : IRequirementTarget
+    {
+        public static readonly Requirement<Options> MustHaveTimeout =
```
[the remark this answers is not in the document]
Comment (FrolYo) [reply src/dotnet/Chat.UI.Blazor/Services/IdleAudioMonitor.cs:182]: changed to simple Validate.

**Pull request merged**